adafruit_cursorcontrol/cursorcontrol_cursormanager.py

```diff
@@ -236,45 +236,103 @@ def _check_cursor_movement(self) -> None:
 
 
 class DebouncedCursorManager(CursorManager):
-    """Simple interaction user interface interaction for Adafruit_CursorControl.
-    This subclass provide a debounced version on the A button and provides queries for when
-    the button is just pressed, and just released, as well it's current state. "Just" in this
-    context means "since the previous call to update."
+    """Simple user interface interaction for Adafruit_CursorControl.
+    This subclass provide a debounced version on the A, B, Start or Select buttons
+    and provides queries for when the buttons are just pressed, and just released,
+    as well their current state. "Just" in this context means "since the previous call to update."
 
     :param Cursor cursor: The cursor object we are using.
     """
 
     def __init__(self, cursor: Cursor, debounce_interval: float = 0.01) -> None:
         CursorManager.__init__(self, cursor)
-        self._debouncer = Debouncer(
-            lambda: bool(self._pad_states & (1 << self._pad_btns["btn_a"])),
-            interval=debounce_interval,
-        )
+        self._debouncers = {}
+        for btn, _ in self._pad_btns.items():
```

Comment:
@tekktrik thanks for review. I did try with `keys()` and the pylint returned the following warning ( just run on my WSL ubuntu 20.04 the following command `pre-commit run --all`)

> pylint (library code)....................................................Failed
> - hook id: pylint
> - exit code: 16
> 
> ************* Module adafruit_cursorcontrol.cursorcontrol_cursormanager
> adafruit_cursorcontrol/cursorcontrol_cursormanager.py:250:19: C0201: Consider iterating the dictionary directly instead of calling .keys() (consider-iterating-dictionary)
>

Reply:
I guess I could do something like this too: `#pylint: disable=consider-iterating-dictionary`

Reply:
Ahh, thanks for the error info!  What it's saying is that there isn't a need to use `keys()` at all because you can just iterate the dictionary keys using `in` and still avoid getting the value:

```python
for btn in self._pad_btns:
    ...
```

Try that and see if that gets rid of the error you mentioned?

Reply:
@tekktrik it works and the pylint error disappeared. thanks !